application/src/Api/Representation/AbstractResourceEntityRepresentation.php

```diff
@@ -526,18 +544,32 @@ public function displayTitle($default = null)
      * Get the display description for this resource.
      *
      * @param string|null $default
+     * @param string|null $lang
      * @return string|null
      */
-    public function displayDescription($default = null)
+    public function displayDescription($default = null, $lang = null)
     {
+        $description = null;
         $template = $this->resourceTemplate();
         if ($template && $template->descriptionProperty()) {
             $descriptionTerm = $template->descriptionProperty()->term();
         } else {
             $descriptionTerm = 'dcterms:description';
         }
 
-        $description = (string) $this->value($descriptionTerm, ['default' => $default]);
+        if ($lang !== null) {
+            if ($descriptionValue = $this->value($descriptionTerm, ['default' => $default, 'lang' => $lang])) {
+                if (!empty($descriptionValue->valueResource())) {
```

Comment:
This is a new change, right?

The prior code would just print URIs for "descriptions" that were Resource type... (since that's what the toString for a resource does). This PR should maintain that same behavior: even if we wanted to change it, we'd want to change it for both the "filter locale" and "non filter" paths simultaneously.

You pretty much don't need any of this extra logic in that case.

Reply:
Actually before this commit. this was not printing URIs nor anything,the description ended up blank.  Which seemed a little curious to me. Let me check this back, I’m wondering if this is something to do with my « values resources » being CustomVocab data types in my test data.

Reply:
If it's a problem specifically with CustomVocab "resource" values, that probably indicates an issue with CustomVocab rather than with this code.